### ISSUE TITLE
[Suggestion] [Rails] Coal loader/ Way to store coal in steam locomotive

### ISSUE THREAD

**Issue by slava110** [Cassiobsk8/Industrial_Renewal#6]
**Feature you want:**
Any way to store coal in steam locomotive ; Something that can increase duration - block for railroad stations like coal loader etc. Automatization! Yes, we can use activator blocks from other mods, but it isn't good for big railroad systems I think.

**Comment by Cassiobsk8** [comment]
Yes it is in my plans, I just left the locomotive a little bit because I spent 5 months learning how to render it there when I did I went to other blocks.
but I'll be back in it probably tomorrow.
